config/locales/exceptions.en.yml

```diff
@@ -522,6 +522,12 @@ en:
         code: 'VAOS_403'
         detail: 'This user is forbidden from performing requests on the upstream server'
         status: 403
+      VAOS_404:
+        <<: *external_defaults
+        title: Not Found
+        code: 'VAOS_404'
+        detail: 'The resource could not be found'
+        status: 404
```

Comment:
Question, why was this needed?

Reply:
might be another way around this but without the specific 404 entry here it was being mapped to a 400 by default